src/DataTable/DataTable.svelte

```diff
@@ -499,7 +503,7 @@
               {#if !nonSelectableRowIds.includes(row.id)}
                 {#if radio}
                   <RadioButton
-                    name="select-row-{row.id}"
+                    name="{id}-{row.id}"
                     checked={selectedRowIds.includes(row.id)}
                     on:change={() => {
                       selectedRowIds = [row.id];
```

Comment:
If we can, I suggest taking this opportunity to provide a solution that affords progressive enhancement:

- Have the `name` attribute be defined by the component consumers; perhaps exposed as a prop, that way every Radio button in the table has a matching `name` to be gathered server-side via `formData.get('school')`
- Have the `value` attribute match the given Row's ID, available in `row.id`

In this way, a DataTable can be wrapped in `<form>` and be submittable without JS.

Reply:
Good call. You're right – the `name` should in fact be the same for all rows in the table (barring the select all).